qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -367,9 +367,55 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def particle_number(n_orbitals, mapping="jordan_wigner"):
+    r"""Computes the particle number operator :math:`\hat{N}=\sum_\alpha \hat{n}_\alpha` in
+    the Pauli basis. Its eigenvalue is the number of particles in a given state of the
```

Comment:
Using "a given state" is a bit awkward since most states don't have a definite number of particles. Maybe "Its eigenstates are states with a definite number of particles given by their corresponding eigenvalue"? Or just remove the sentence

Reply:
Better to remove the sentence.